language-server/src/main/java/org/sonarlint/languageserver/SonarLintLanguageServer.java

```diff
@@ -126,7 +121,7 @@
   private Map<String, String> analyzerProperties;
   private int ruleServerPort;
 
-  public SonarLintLanguageServer(InputStream inputStream, OutputStream outputStream) throws IOException {
+  public SonarLintLanguageServer(InputStream inputStream, OutputStream outputStream, Collection<URL> analyzers) throws IOException {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the declaration of thrown exception 'java.io.IOException', as it cannot be thrown from constructor's body. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)